tests/playwright/sanity/assets/dev/js/editor/elements/views/widget.test.js

```diff
@@ -0,0 +1,16 @@
+const { test, expect } = require( '@playwright/test' );
+const WpAdminPage = require( '../../../../../../../pages/wp-admin-page' );
+
+test( 'Check if the empty placeholder is displayed in the Image Carousel', async ( { page }, testInfo ) => {
+	// Arrange.
+	const wpAdmin = new WpAdminPage( page, testInfo ),
+		editor = await wpAdmin.useElementorCleanPost();
```

Comment:
Can you add all the widgets that need to be verified?

Reply:
@TzviRabinovitch 
That could be like 20 widgets.